mc-react-components/src/lib/date-range-picker/stories/DateRangePicker.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { DateTimeRangePicker } from '../date-range-picker';
+
+export default {
+  title: 'Range Picker',
+  component: DateTimeRangePicker,
+};
+
+export const DateTime = () => <DateTimeRangePicker onChange={() => null}/>;
```

Comment:
You can add an action on every property that is an event/function:
```
export const DateTime = () => <DateTimeRangePicker onChange={action('date change')}/>;
```
leads to 
![image](https://user-images.githubusercontent.com/12687466/88939860-b52cf480-d28f-11ea-91a4-63c540100a74.png)

Reply:
done